Modules/socketmodule.h

```diff
@@ -342,7 +342,8 @@ typedef struct {
 
 */
 
-/* C API for usage by other Python modules */
+/* C API for usage by other Python modules.
+ * Always add new things to the end for binary compatibility. */
```

Comment:
I am not sure this note is correct or not. The binary compatibilty also depends how to use it, no?

Reply:
Sure it does. I just copied the comment from the .c file. I figured it made more sense together with the definition in the header file. However, I don't think this comment is really needed. I'd recommend just deleting it.

Reply:
In case of doubt, I prefer to keep it. It makes sense to only add new members at the end. If a C extension is built with Python 3.10 and a new filed is added to Python 3.11, the Python 3.10 remains ABI compatible. The comment looks correct.